src/actions/createCategoryAction.ts

```diff
@@ -14,11 +13,11 @@ export const createCategoryAction = async (state: unknown, data: FormData) => {
 
 	const db = await withDatabase();
 
-	await new CategoryController(db).create({
+	const result = await new CategoryController(db).create({
 		name: validateData.data?.name,
 		video: undefined,
 	});
 
-	revalidateTag('./');
-	redirect('/');
+	revalidateTag('result');
```

Comment:
Otestoval jsi si, že se to revaliduje?

Reply:
no když se vrátím na main page, tak v netvorku se vráti všechny ty image, tak že je to asi ok?